pep-0581.rst

```diff
@@ -105,25 +114,29 @@ Issues with Roundup / bpo
 - Creating an account has been a hassle. There have been reports of people
   having trouble creating accounts or logging in.
 
-Why not GitLab
---------------
+
+Why not GitLab?
+===============
 
 Had we migrated to GitLab instead of GitHub in 2017, this PEP would have been
 titled "Using GitLab Issues for CPython".
 
-Why not another issue tracker
------------------------------
+
+Why not another issue tracker?
+==============================
 
 Using another issue tracker will require yet another learning curve, for having
 to learn and get used to a different interface. We'll also need to learn and
 figure out how to build the integrations with GitHub.
 
-By using GitHub issues, where the CPython source code is currently hosted and where
-pull requests are taking place, we'll be providing consistent experience to
-contributors and maintainers, while not having to jump from one interface to another.
+By using GitHub issues, where the CPython source code is currently
+hosted and where pull requests are taking place, we'll be providing
+consistent experience to contributors and maintainers, while not
```

Comment:
We are not the only project that uses a different issue tracker while using GitHub to host their source code. Some projects (SQLAlchemy) even use GitLab for issue tracking.